src/components/central-program/staff-roles-table.js

```diff
@@ -15,12 +15,19 @@ const createFirstRow = data => {
   }, initialObject)
 }
 
+const columnsFormatter = (cell, row, rowIndex, formatExtraData) => {
+  if (rowIndex === 0) {
+    return <span className="strong">{row.role_description}</span>
```

Comment:
Hey @MorkWood2 , sorry for the very slow review on this. I had to look up how to checkout a branch from a forked repo 😂 . 

The style for the Total looks good, but the names of the staff roles aren't showing up in the table for me anymore. I think this if statement means we only put text in the cell if it's the first row? Maybe we just need to add the strong if it's the first row?

Reply:
Right on! It's been a while, I'll check it out and see where I went wrong. Thanks!